### DISCUSSION TITLE
Export and Import data based on the query

### DISCUSSION THREAD

**Discussion by shrivatsahosabettu** [dbeaver/dbeaver#35842] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Is there any option or I would like to have a option to export and import data based on the query.
Currently if I need to do data transfer either I have to unload in one of the format of csv,xlsx, insert SQL, json etc.

Instead of this if there is option to transfer data based on the query with WHERE condition, that will be good. It will be nice I don't want to go through 2 step process of export in xls/csv/sql statements and then import.

**Comment by E1izabeth** [comment, marked as answer]
Hi, @shrivatsahosabettu 
You can select your query or put a cursor to the query you need, call the context menu, and choose `Export form Query`
![image](https://github.com/user-attachments/assets/a510168c-02c3-4f13-aa47-ae9b7ee0883e)

Then you can choose export to Database
![image](https://github.com/user-attachments/assets/39f14ec7-efed-41f9-859d-9ebb72333a94)

Is that what you need?

**Comment by shrivatsahosabettu** [comment]
@E1izabeth Yes this is exactly what I am looking for. Thanks for explaining with screen prints.

**Comment by E1izabeth** [comment]
Yes, it is

**Comment by shrivatsahosabettu** [comment]
Thank you @E1izabeth, you can close this one then.

**Comment by quangbilly79** [comment]
What about Import setting?
